resources/SettingsTabComponents/SettingsTable.qml

```diff
@@ -241,10 +254,7 @@ Rectangle {
             model: TableModel {
                 id: tableModel
 
-                rows: [{
-                        "Name": "",
-                        "Value": ""
-                    }]
+                rows: []
```

Comment:
Why do we need to remove this?

Reply:
this creates an empty row here:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/35755741/229944499-cee42fb5-9dfd-4982-8790-3dbef2ec88df.png">

Reply:
removing it makes it like this:
<img width="341" alt="image" src="https://user-images.githubusercontent.com/35755741/229944604-71805992-dd2f-4ef3-928a-b2a25cb5d7e7.png">

Reply:
👍